.travis.yml

```
@@ -0,0 +1,14 @@
+dist: bionic
+language: node_js
+node_js:
+  - 8.12.0
+services:
+  - docker
+before_install:
+  - docker run -d -p 4444:4444 -p 5900:5900 -v /dev/shm:/dev/shm --name selenium selenium/standalone-chrome-debug
+before_script:
+  - cd parijat-frontend
+  - yarn
+  - yarn dev &
+  - sleep 30
```

```suggestion
  - until curl -s localhost:8080; do sleep 5; done
```

Better to wait until the site is up. @jasson99 what do you think?